src/pages/ProfilePage/components/EditProfileModal/EditProfileModal.scss

```diff
@@ -23,15 +26,26 @@
     background: #ffffff;
     padding: 2rem;
     border-radius: 0.5rem;
+    border: 0.2rem solid $MVP-black;
     box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1),
       0 2px 4px -2px rgba(0, 0, 0, 0.1);
     width: 100%;
     max-width: 600px;
     max-height: 90vh;
     overflow-y: auto;
     z-index: 1001;
+
+    &::-webkit-scrollbar {
+      width: 0;
+      height: 0;
+    }
+
+    // Hide scrollbar for other browsers
+    -ms-overflow-style: none; // IE and Edge
+    scrollbar-width: none; // Firefox
   }
```

Comment:
I got this deprecation warning for this part of the file: "Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming version. To keep the existing behavior, move the declaration above the nested rule. To opt into the new behavior, wrap the declaration in `& {}`."

To make sure we avoid any issues in the future, you can wrap declarations in '& {}'

&::-webkit-scrollbar {
  width: 0;
  height: 0;
}
& {
  -ms-overflow-style: none; // IE and Edge
  scrollbar-width: none; // Firefox
}

or simply move the declarations above the nested rule.